### PR TITLE
update(README.md): tweaks

### PR DESCRIPTION
**Pull request by ACoolmanTelicent**
Regarding: https://github.com/Telicent-io/rdf-libraries/issues/37

Note: Generated docs will NOT appear until a github admin can do: https://github.com/telicent-oss/rdf-libraries/issues/24

### CODE REVIEW

**Review by davec504 — APPROVED** [review]

File: packages/OntologyService/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Simple client-side library for working with RDF ontologies. Includes basic CRUD
 abilities and functions to help you navigate an ontology hierarchy.
 
-For further information on how to use the library see the [API Documentation](https://somehting.com)
+For further information on how to use the library see the [API Documentation](README.md#API)
```
Comment (dptelicent) [inline packages/OntologyService/README.md:6]:
```suggestion
For further information on how to use the library, see the [API Documentation](README.md#API).
```

File: packages/RdfService/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 RdfService is a helper library that abstracts away the complexity of interacting with RDF triplestores, providing basic CRUD abilities.
 
-For futher information on how to use the library see the [API documentation](https://needtocreate.com)
+For futher information on how to use the library see the [API documentation](README.md#API)
```
Comment (dptelicent) [inline packages/RdfService/README.md:5]:
```suggestion
For futher information on how to use the library, see the [API documentation](README.md#API).
```

File: README.md [hunk 3]
```diff
@@ -2,15 +2,21 @@
...
+This monorepo's sub-packages are published to https://www.npmjs.com/org/telicent-oss:
+
 ```sh
-cd <yourProject>
 yarn install @telicent-oss/ontologyservice
 ```
 
 ## Usage
 
+A simple example:
```
Comment (procolecato) [inline README.md:19]: That does what?  I'm guessing from the code fragment that it gets all diagrams from the ontology service.  If I'm correct then this line should read:
"A simple example to get all diagrams from the ontology service:"

File: README.md [hunk 4]
```diff
@@ -19,74 +25,60 @@ const ontologyService = new OntologyService("http://localhost:3030/", "ontology"
...
 - WARNING: `import x from '.'` can cause problems. Instead use `import x from './index'`
 - If changing code then all commands must be run via nx else it will use the old code in `node_modules`
-- nx commands can be run from any monorepo package and will resolve as if run on monorepo root
-- More nx documentation at brilliant AI bot https://nx.dev/ai-chat
+- nx commands can be run from any sub-directory and will resolve as if run on monorepo root
+- More nx documentation at: https://nx.dev/ai-chat
+
+</details>
+
+## API
```
Comment (procolecato) [inline README.md:80]: I'll review the links in here once the API docs are added.

File: packages/OntologyService/README.md [hunk 5]
```diff
@@ -3,7 +3,7 @@
...
 
 ## Install
```
Comment (procolecato) [inline packages/OntologyService/README.md:8]: As per the main README.md, in the Usage section, below, the code example just needs a one-liner to introduce it, e.g. "A simple example to get all diagrams from the ontology service:".

File: packages/RdfService/README.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
...
 
 ## Install
```
Comment (procolecato) [inline packages/RdfService/README.md:7]: As per the main README.md and the OntologyService README.md, in the Usage section, below, the code example just needs a one-liner to introduce it, e.g. "A simple example that runs a query on the 'ontology' rdf service:".

File: packages/RdfService/README.md [hunk 7]
```diff
@@ -19,3 +19,6 @@ const rdfService = new RdfService("http://localhost:3030/", "ontology");
 const diagrams = await rdfService.runQuery(query);
```
Comment (procolecato) [inline packages/RdfService/README.md:19]: What is the 'query' that is being run here?  Is it one to get diagrams from the 'ontology' rdfService?  Or is it doing something else (in which case the destination variable should be named to something other than 'diagrams').

**Review by procolecato — COMMENTED** [review]
I don't know if this review is still required as this branch has already been merged but here goes anyway ...

**Pull request merged**